frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1274,6 +1274,7 @@
                 };
 
                 utilities.sendRequest(parameters);
+                $location.path("web/dashboard");
```

Comment:
Can you please use `$state.go()` instead of using `$location.path`?

Reply:
done using state, please review https://github.com/Cloud-CV/EvalAI/pull/2106/commits/680f204cfce05749883b8c0ddb4efdb86d76e31a

Reply:
https://github.com/ArshdeepSingh98/EvalAI/pull/1/commits/73d87a065151460c0372414f12d8ec7684ff656d
Please review,
Thanks